README.md

```diff
@@ -116,6 +116,16 @@ that improve the experience for consumers on these platforms.
 | OpenBSD   | x86-64      |
 | FreeBSD   | x86-64      |
 
+### FIPS Compliance
+
+AWS-LC can be built in FIPS mode for compliance with FIPS 140-3 requirements. However, please note that:
+
+- Static FIPS builds are only supported on Linux platforms
```

Comment:
Same as above, we should link to the FIPS.md (https://github.com/aws/aws-lc/blob/main/crypto/fipsmodule/FIPS.md)